app/src/main/java/co/copperhead/pdfviewer/PdfViewer.java

```diff
@@ -21,18 +21,24 @@
 import android.widget.NumberPicker;
 
 public class PdfViewer extends Activity {
+    private static final int MIN_ZOOM_LEVEL = 1;
     private static final int MAX_ZOOM_LEVEL = 4;
+    private static final int ALPHA_LOW = 130;
+    private static final int ALPHA_HIGH = 255;
     private static final int ACTION_OPEN_DOCUMENT_REQUEST_CODE = 1;
     private static final String STATE_URI = "uri";
     private static final String STATE_PAGE = "page";
     private static final String STATE_ZOOM_LEVEL = "zoomLevel";
 
+    private boolean mButtonDisabled;
```

Comment:
mZoomButtonDisabled would be clearer, but I think you can do without this field. It has the information it needs without tracking the state.